test/python/scenarios.json

```diff
@@ -0,0 +1,31 @@
+{
+    "install_additional_python": {
+        "image": "ubuntu:focal",
+        "features": {
+            "python": {
+                "version": "3.10.5",
+                "additionalVersions": "3.8,3.9.13"
+            }
+        }
+    },
+    "install_jupyterlab": {
+        "image": "mcr.microsoft.com/vscode/devcontainers/base:focal",
+        "remoteUser": "vscode",
+        "features": {
+            "common-utils": {
```

Comment:
Curious, should this be in global tests as this scenario is referencing `common-utils` and `python` Features ?

Reply:
IMO it's a judgement call.  If the primary point of the test is testing `python` _with_  another feature, it is cool to be placed under `test/python`. 

Right now the global tests don't run on PRs (might wanna do that tho). I'll leave it here for at least the time being.

Reply:
> IMO it's a judgement call. If the primary point of the test is testing python with another feature, it is cool to be placed under test/python.

Good to know 👍